packages/server/webpack.common.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+
+module.exports = {
+    entry: './src/server.ts',
+    target: 'node',
+    output: {
+        filename: 'server.js',
+        path: path.resolve(__dirname, 'dist'),
+    },
+    resolve: {
+        extensions: ['.ts', '.js'],
+    },
+    module: {
+        rules: [
+            { test: /\.ts$/, use: { loader: 'ts-loader', options: { projectReferences: true } } },
+        ],
+    },
+    ignoreWarnings: [
+        {
+            module: /node_modules\/express\/lib\/view\.js/,
+            message: /the request of a dependency is an expression/,
+        },
+        {
+            module: /node_modules\/log4js\/lib\/appenders\/index\.js/,
+            message: /the request of a dependency is an expression/,
+        },
+    ],
```

Comment:
From searching online these don't seem like big issues. The advice was either don't bundle them or ignore the errors. The reason for adding `webpack` was so we could bundle everything into a single file, so I think we just have to ignore them